pkg/collector/corechecks/gpu/gpu.go

```diff
@@ -154,6 +154,10 @@ func (m *Check) emitSysprobeMetrics(snd sender.Sender) error {
 	for pid, pidStats := range stats.ProcessStats {
 		// Per-PID metrics are subject to change due to high cardinality
 		tags := []string{fmt.Sprintf("pid:%d", pid)}
+		if pidStats.Metadata.ContainerID != "" {
+			tags = append(tags, fmt.Sprintf("container_id:%s", pidStats.Metadata.ContainerID))
```

Comment:
I wonder how useful this tag is by itself, instead of using the containerID to look for all of the container tags from the tagger?

Reply:
We're removing this part from this PR, we'll add the container tags later using the tagger as you say.